subprojects/core/src/integTest/groovy/org/gradle/caching/configuration/internal/BuildCacheCompositeConfigurationIntegrationTest.groovy

```diff
@@ -121,7 +121,8 @@ class BuildCacheCompositeConfigurationIntegrationTest extends AbstractIntegratio
         }
 
         def finalizeOps = operations.all(FinalizeBuildCacheConfigurationBuildOperationType)
-        finalizeOps.size() == expectedCacheDirs.size()
+        def opsPerCache = configCache ? 2 : 1
+        finalizeOps.size() == expectedCacheDirs.size() * opsPerCache
```

Comment:
As mentioned in the description, 2 of these build ops are fired in a CC miss build - one for the store operation and one for the load operation.